docs/references/fluxctl.md

```diff
@@ -224,6 +224,7 @@ Available Commands:
   version        Output the version of fluxctl
 
 Flags:
+      --context string                  The kubectl context to use
```

Comment:
```suggestion
      --context string                  The kubeconfig context to use
```

Reply:
to follow kubectl semantics i suggest not adding k8s-